content/en/docs/concepts/services-networking/service.md

```diff
@@ -203,10 +203,13 @@ described in detail in [EndpointSlices](/docs/concepts/services-networking/endpo
 
 ### Application protocol
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+The `appProtocol` field provides a way to specify an application protocol for
+each Service port. The value of this field is mirrored by corresponding
+Endpoints and EndpointSlice resources.
```

Comment:
(nit)
```suggestion
Endpoint and EndpointSlice objects.
```

Reply:
Updated to use "objects", not as sure about "Endpoint". It looks like we've more consistently used "Endpoints" throughout docs from what I can tell. It's definitely confusing to have a plural resource name, but the alternative of "Endpoint" also feels confusing. Not sure what's best here but for now I've left it as "Endpoints".

Reply:
D'oh. The `kind` is _Endpoints_ plural isn't it. That was the right call.

Reply:
Haha yeah, it is super confusing, especially alongside the singular EndpointSlice name.